src/main/webapp/dev/js/main/instructorCourseEdit.es6

```diff
@@ -473,6 +473,20 @@ $(document).ready(() => {
         showNewInstructorForm();
     });
 
+    $("[name='instructorisdisplayed']").change((e) => {
+        if ($(e.target).prop('checked')) {
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('readonly', false);
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('value', 'Instructor');
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input')
+                    .attr('placeholder', 'E.g.Co-lecturer, Teaching Assistant');
+        } else {
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('readonly', false);
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('value', 'Instructor');
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input')
+                    .attr('placeholder', '(This instructor will NOT be displayed to students)');
```

Comment:
Apologies once again, have changed this to set the value to an empty string if the user unchecks, allowing the placeholder to display.